index.js

```diff
@@ -1505,6 +1505,29 @@
   }
   S.pipe = def('pipe', {f: [Z.Foldable]}, [f(Fn($.Any, $.Any)), a, b], pipe);
 
+  //# pipeK :: (Foldable f, Chain m) => f (Any -> m Any) -> m a -> m b
+  //.
+  //. Takes an array of functions assumed to be unary which return
+  //. values with a [Chain][], and a value of that Chain, and returns the
+  //. result of applying the sequence of transformations to the initial value.
+  //.
+  //. In general terms, `pipeK` performs left-to-right [Kleisli][] composition
+  //. of an array of functions. `pipeK([f, g, h], x)` is equivalent to
+  //. `chain(h, chain(g, chain(f, x)))`.
```

Comment:
Let's replace “an array of functions” with “a sequence of functions” as we did in #473.

Reply:
Fixed